src/gui/BandSplitter/BandSplitterPlot.cpp

```diff
@@ -160,10 +177,11 @@ BandSplitterPlot::~BandSplitterPlot()
 
 void BandSplitterPlot::updateCutoffFrequency()
 {
-    for (int bandIndex = 0; bandIndex < numBands; ++bandIndex) //bands 0, 1, 2, 3
+    for (int bandIndex = 0; bandIndex < numBands; ++bandIndex) //bands 0, 1, 2, 3, 4, 5
     {
-        //bands 1 & 2 assigned cutoff 1, bands 2 & 3 assigned cutoff 2 - this will be the current cutoff frequency in Hz
-        const auto& cutoffParam = bandIndex < (numBands / 2) ? bandSplitterParams.cutoff : bandSplitterParams.cutoff2;
+        //bands 1 & 2 assigned cutoff 1, bands 2 & 3 assigned cutoff 2, bands 5 & 6 assigned cutoff 3 - this will be the current cutoff frequency in Hz
+        const auto& cutoffParam = bandIndex < (numBands / 3) ? bandSplitterParams.cutoff : bandIndex <= numBands / 2 ? bandSplitterParams.cutoff2
+                                                                                                                     : bandSplitterParams.cutoff3;
```

Comment:
It might be easier to do this with an "immediately invoked lambda":
```cpp
const auto& cutoffParam = []
{
    if (bandIndex < (numBands / 3))
        return bandSplitterParams.cutoff;
    // ...
}();
```

Reply:
Also, would we want the same for line 220 in this file?

Reply:
I think capturing should work like:
```cpp
const auto& cutoffParam = [this]
{
    // ...
}();
```
Or if you want to be more conservative, I think this should also work:
```cpp
const auto& cutoffParam = [&bandSplitterParams = this->bandSplitterParams]
{
    // ...
}();
```
Yeah, it looks line 220 has the same pattern... if we could combine the logic into a single lambda that would be even better!